src/Equinox.CosmosStore/CosmosStore.fs

```diff
@@ -565,33 +565,43 @@ module internal Sync =
 
 module Initialization =
 
-    type [<RequireQualifiedAccess>] Provisioning = Container of rus: int | Database of rus: int | Serverless
-    let adjustOfferC (c:Container) (rus : int) = async {
+    // Note: the Cosmos SDK does not (currently) support changing the Throughput mode of an existing Database or Container.
+    type [<RequireQualifiedAccess>] Throughput = Manual of rus: int | Autoscale of maxRus: int
+    type [<RequireQualifiedAccess>] Provisioning = Container of Throughput | Database of Throughput | Serverless
+    let toThroughputProperties = function
+        | Some (Throughput.Manual rus) -> ThroughputProperties.CreateManualThroughput(rus)
+        | Some (Throughput.Autoscale maxRus) -> ThroughputProperties.CreateAutoscaleThroughput(maxRus)
+        | _ -> null
```

Comment:
Please change this from `Throughput option -> ThroughputProperties` to `Throughput -> ThroughputProperties`. 
Although `ReplaceThroughputAsync` accepts `null` as a parameter, leaving nullable values out there isn't good.